ios/sdk/sdk.xcodeproj/project.pbxproj

```diff
@@ -14,6 +14,9 @@
 		0B93EF7E1EC9DDCD0030D24D /* RCTBridgeWrapper.h in Headers */ = {isa = PBXBuildFile; fileRef = 0B93EF7C1EC9DDCD0030D24D /* RCTBridgeWrapper.h */; };
 		0B93EF7F1EC9DDCD0030D24D /* RCTBridgeWrapper.m in Sources */ = {isa = PBXBuildFile; fileRef = 0B93EF7D1EC9DDCD0030D24D /* RCTBridgeWrapper.m */; };
 		0BA13D311EE83FF8007BEF7F /* ExternalAPI.m in Sources */ = {isa = PBXBuildFile; fileRef = 0BA13D301EE83FF8007BEF7F /* ExternalAPI.m */; };
+		0BB9AD771F5EC6CE001C08DB /* CallKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 0BB9AD761F5EC6CE001C08DB /* CallKit.framework */; settings = {ATTRIBUTES = (Weak, ); }; };
+		0BB9AD791F5EC6D7001C08DB /* Intents.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 0BB9AD781F5EC6D7001C08DB /* Intents.framework */; settings = {ATTRIBUTES = (Weak, ); }; };
```

Comment:
* Because `CallKit.framework` and `Intent.framework` are `@improt`ed, it's unnecessary to reference them anywhere in this file.

* `Intents.framework` was `@import`ed in the app, yet its reference in a `project.pbxproj` was in the sdk.

Reply:
Do you know if that works on iOS 9? If it doesn, great!

Reply:
`@import` is about build time, not about run time so there's no concern about iOS 9. Besides, `@import` aka modules were introduced around iOS 7.